imageroot/systemd/user/rspamd.service

```
@@ -25,6 +25,7 @@ ExecStart=/usr/bin/podman run \
     --volume=rspamd-override:/etc/rspamd/override.d:z \
     --volume=/dev/log:/dev/log \
     ${MAIL_RSPAMD_IMAGE}
+ExecStartPost=/usr/bin/bash -c "while ! exec 3<>/dev/tcp/127.0.0.1/11334 &>/dev/null; do sleep 3 ; done ; printf 'GET /ping\r\n\r\n' >&3"
```

Maybe a sleep of 1 second could catch earlier a running instance?
```suggestion
ExecStartPost=/usr/bin/bash -c "while ! exec 3<>/dev/tcp/127.0.0.1/11334 &>/dev/null; do sleep 1 ; done ; printf 'GET /ping\r\n\r\n' >&3"
```